tests/jenkins/TestDataPrepperReleaseArtifacts.groovy

```diff
@@ -88,4 +103,18 @@ class TestDataPrepperReleaseArtifacts extends BuildPipelineTest {
         assertThat(actualS3Upload, notNullValue())
         assertThat(actualS3Upload.args[0].get('path'), equalTo("data-prepper/${version}/"))
     }
+
+    def getCurlCommands() {
+        runScript('jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile')
```

Comment:
I don't think the `runScript` should be in this function. This is is where we actually execute the test.